sandbox/src/mocks/r4/sendQuestionnaireResponse/requests/basicQuestionnaireResponse.json

```diff
@@ -0,0 +1,42 @@
+{
+  "resourceType": "QuestionnaireResponse",
+  "questionnaire": "https://fhir.nhs.uk/Questionnaire/ERS-ShortList",
+  "basedOn":  [
+    {
+      "reference": "ServiceRequest/a.832db7fa-ebdd-44b6-ab3b-8329c2d43149/_history/5"
+    }
+  ],
+  "subject": {
+    "identifier": {
+      "system": "https://fhir.nhs.uk/Id/nhs-number",
+      "value": "9912003888"
+    },
+  },
+  "status" : "completed",
+  "authored" : "2022-01-11T16:40:00+00:00",
+  "item":  [
+    {
+      "linkId": "searchCriteria",
+      "answer":  [
+        {
+          "valueString": "HealthcareService?supportedSpecialty=general-medicine&supportedPriority=routine"
+        }
+      ]
+    },
+    {
+      "linkId": "services",
+      "answer":  [
+        {
+          "valueReference": {
+            "reference": "HealthcareService/76de72bc-f001-49fa-84c6-af15a4fdb72c"
+          }
+        },
+        {
+          "valueReference": {
```

Comment:
Currently e-RS effectively only supports a single service on the shortlist. We should probably keep the examples aligned with this.

(I have heard talk about allowing multiple services in the conversation so we should keep the flexibility of having the shortlist for A&G)